ui/feral_druid/apls/p2.apl.json

```diff
@@ -1,19 +1,23 @@
-    {"action":{"condition":{"and":{"vals":[{"cmp":{"op":"OpLe","lhs":{"auraRemainingTime":{"sourceUnit":{"type":"CurrentTarget"},"auraId":{"spellId":17392,"rank":4}}},"rhs":{"const":{"val":"8.0"}}}},{"cmp":{"op":"OpLe","lhs":{"auraRemainingTime":{"sourceUnit":{"type":"CurrentTarget"},"auraId":{"spellId":9907,"rank":4}}},"rhs":{"const":{"val":"8.0"}}}}]}},"castSpell":{"spellId":{"spellId":17392,"rank":4}}}},
+    {"action":{"condition":{"and":{"vals":[{"cmp":{"op":"OpLe","lhs":{"auraRemainingTime":{"sourceUnit":{"type":"CurrentTarget"},"auraId":{"spellId":17392,"rank":4}}},"rhs":{"const":{"val":"14.0"}}}},{"cmp":{"op":"OpLe","lhs":{"auraRemainingTime":{"sourceUnit":{"type":"CurrentTarget"},"auraId":{"spellId":9907,"rank":4}}},"rhs":{"const":{"val":"14.0"}}}}]}},"castSpell":{"spellId":{"spellId":17392,"rank":4}}}},
     {"action":{"condition":{"not":{"val":{"auraIsActive":{"auraId":{"spellId":768}}}}},"castSpell":{"spellId":{"itemId":10646}}}},
     {"action":{"condition":{"and":{"vals":[{"not":{"val":{"auraIsActive":{"auraId":{"spellId":768}}}}},{"cmp":{"op":"OpLe","lhs":{"math":{"op":"OpAdd","lhs":{"currentMana":{}},"rhs":{"const":{"val":"1500.0"}}}},"rhs":{"math":{"op":"OpDiv","lhs":{"currentMana":{}},"rhs":{"currentManaPercent":{}}}}}}]}},"castSpell":{"spellId":{"itemId":12662}}}},
     {"action":{"condition":{"and":{"vals":[{"not":{"val":{"auraIsActive":{"auraId":{"spellId":768}}}}},{"cmp":{"op":"OpLe","lhs":{"math":{"op":"OpAdd","lhs":{"currentMana":{}},"rhs":{"const":{"val":"2250.0"}}}},"rhs":{"math":{"op":"OpDiv","lhs":{"currentMana":{}},"rhs":{"currentManaPercent":{}}}}}}]}},"castSpell":{"spellId":{"otherId":"OtherActionPotion"}}}},
+    {"action":{"condition":{"and":{"vals":[{"not":{"val":{"auraIsActive":{"auraId":{"spellId":768}}}}},{"cmp":{"op":"OpLe","lhs":{"currentManaPercent":{}},"rhs":{"const":{"val":"80%"}}}},{"not":{"val":{"or":{"vals":[{"and":{"vals":[{"spellIsKnown":{"spellId":{"itemId":12662}}},{"spellIsReady":{"spellId":{"itemId":12662}}}]}},{"and":{"vals":[{"spellIsKnown":{"spellId":{"otherId":"OtherActionPotion"}}},{"spellIsReady":{"spellId":{"otherId":"OtherActionPotion"}}}]}}]}}}}]}},"castSpell":{"spellId":{"otherId":"OtherActionDefensiveEquip"}}}},
```

Comment:
I think this still needs to be updated from OtherActionDefensiveEquip to specifically RoM, right?

Reply:
Yep, fixed this.